llvm/include/llvm/BinaryFormat/COFF.h

```diff
@@ -726,7 +727,10 @@ enum BaseRelocationType : unsigned {
   IMAGE_REL_BASED_HIGHADJ = 4,
   IMAGE_REL_BASED_MIPS_JMPADDR = 5,
   IMAGE_REL_BASED_ARM_MOV32A = 5,
+  IMAGE_REL_BASED_RISCV_HI20 = 5,
```

Comment:
```suggestion
  IMAGE_REL_BASED_RISCV_HIGH20 = 5,
```

Reply:
Looks like IMAGE_REL_BASED_RISCV_HI20 already appears in other code according to google.